make/platformio.sconscript

```diff
@@ -4260,8 +4260,16 @@ source_files = []
 for src in env.LookupSources(variant_dir, src_dir, True, src_filter):
     source_files.append(env.Object(src))
 
+# get settings file path
+settings_ini_path = ARGUMENTS.get('SETTINGS_PATH', "").decode('base64')
+if settings_ini_path:
+    settings_ini_file = env.File(settings_ini_path, env['PROJECT_DIR'])
+    settings_ini_path = str(settings_ini_file)
```

Comment:
Merge the two lines into one by calling `str()` on the file object on the same line.